### PR TITLE
SN-2: add run-migration.sh with init-database.sh shell-scripts and sql scripts

### PR DESCRIPTION
**Pull request by DarkReduX**

### CODE REVIEW

File: migrations/init_database.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+create table profile
```
Comment (VladislavSenkevichKanda) [inline migrations/init_database.sql:1]: Somewhere you use capslock for CREATE, somewhere you don't. Please use the same typographical convention for all special words like CREATE, OR, REPLACE, UPDATE, IF, NOT, EXIST etc.

**Review by VladislavSenkevichKanda — CHANGES_REQUESTED** [review]

File: init-database.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# From project root
+cd ./migrations/ || exit
+
+initDatabaseScript=$(basename -- "init_database.sql")
+echo "Start init database from $initDatabaseScript script"
+# listOfDoneMigrationScript=("./history/*")
+
+ errorlog=$(mktemp)
+    trap 'rm -f "$errorlog"' EXIT
+    pwcheck="$(psql postgresql://postgres:a!11111111@localhost:5432/social_network -v ON_ERROR_STOP=1 -f "$initDatabaseScript" 0 < "$errorlog")"
```
Comment (VladislavSenkevich) [inline init-database.sh]: Db url should come from env vars. If there are no required env vars then exit the script

File: migrations/0000_profile_functions.sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+CREATE or replace function get_profile(varchar(32)) returns setof profile as
+$$
+begin
+    return query select * from profile where username = $1;
+end
+$$
+    language 'plpgsql';
+
+CREATE or replace function update_profile(varchar(32), varchar(256), varchar(256), timestamp,
```
Comment (VladislavSenkevich) [inline migrations/0000_profile_functions.sql]: Use next function declaration:

CREATE OR REPLACE FUNCTION update_profile(password profile.password%TYPE ... ) returns ...

If you change the type of a table column, you will not need to change the functions that depend on that column.

File: migrations/init_database.sql (hunk 1 above)
Comment (VladislavSenkevich) [inline migrations/init_database.sql]: Somewhere you use capslock for CREATE, somewhere you don't. Please use the same typographical convention for all special words like CREATE, OR, REPLACE, UPDATE, IF, NOT, EXIST etc.

File: run-migrations.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+echo "Starting migrate process..."
+for item in $listOfMigrationScript; do
+
+  findItem=$(find ./history -name "${item}")
+  findItem=${findItem##*/}
+
+  if [ "$item" != "$findItem" ]; then
+    errorlog=$(mktemp)
+    trap 'rm -f "$errorlog"' EXIT
+    pwcheck="$(psql postgresql://postgres:a!11111111@localhost:5432/social_network -v ON_ERROR_STOP=1 -f "$item" 0 < "$errorlog")"
```
Comment (VladislavSenkevich) [inline run-migrations.sh]: Db url should come from env vars. If there are no required env vars then exit the script

**Review by VladislavSenkevich — CHANGES_REQUESTED** [review]

**Pull request merged**